l10n_ec/README.rst

```diff
@@ -0,0 +1,121 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Ecuador's Account Chart
+=======================
+
+This module adds the basic chart of account of Ecuador.
+    * Works with the specifications of the Superintendencia de Compañías.
+
+Plan de Cuentas NIIF Pymes para Ecuador
+=======================================
+
+Agrega el plan de cuentas para Ecuador.
+    * Utiliza la clasificación de la Superintendencia de Compañías.
+
+Installation
+============
+
+To install this module, you need to:
+
+* Add the module to your addons path.
+* Install the module as usual.
+
+Instalación
+===========
+
+Para instalar este módulo se debe:
+
+* Agregar el módulo al directorio de addons.
+* Instalarlo de manera regular.
+
+Configuration
+=============
+
+This module adds all the accounts as said by the Supercias but:
+    * You should delete the accounts that don't apply to your company.
+    * You should add the accounts that are not mandatory but you need.
+    * Keen in mind that you should use the standar coding, if you don't reports will fail.
```

Comment:
`*  Keep in mind that you should use the standard coding. If you don't, reports will fail.`